src/sortStudents.ts

```diff
@@ -1,16 +1,82 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyOfStudents = [...students];
+  const sortOrder: 1 | -1 = order === 'asc' ? 1 : -1;
```

Comment:
You don't need directly define type here, TS have to understand that value can be only 1 or -1
```suggestion
  const sortOrder = order === 'asc' ? 1 : -1;
```